src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionView.xaml.cs

```diff
@@ -197,10 +219,10 @@ private void ListView_SizeChanged(object sender, SizeChangedEventArgs e)
         private void SortableColumnHeader_SizeChanged(object sender, SizeChangedEventArgs sizeChangedEventArgs)
         {
             // GridViewColumnHeader doesn't handle setting minwidth very well so we prevent it here.
-            if (sizeChangedEventArgs.NewSize.Width <= 60)
+            if (sizeChangedEventArgs.NewSize.Width <= 67)
```

Comment:
how did we come up with a diff of 7? :D

Reply:
It's arbitrary, but the word Installed is a tiny bit wider than Version, so that's why I expanded

Reply:
You inspired me to make this vary based on column name. I also shrunk to 64 since it still looks good at that spacing.